.buildkite/pipeline.yml

```diff
@@ -0,0 +1,16 @@
+---
+env:
+  BUILDKITE_DOCKER_COMPOSE_CONTAINER: build
+  BUILDKITE_DOCKER_COMPOSE_FILE: docker-compose.buildkite.yml
+steps:
+  - wait
```

Comment:
What are these for? Are they required after each task which is a dependancy of those after it?